### PR TITLE
Captcha

### PR DESCRIPTION
**Pull request by masteriza**
Please check and confirm the my pool request.

### CODE REVIEW

File: web/pom.xml [hunk 1]
```diff
@@ -152,6 +152,20 @@
             <version>5.5.11</version>
         </dependency>
 
+        <!--Lombok-->
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>1.16.16</version>
+            <scope>provided</scope>
```
Comment (backumich) [inline web/pom.xml:160]: why provided?

File: web/src/main/java/ua/com/vertex/controllers/exceptionHandling/AppErrorController.java [hunk 2]
```diff
@@ -2,19 +2,12 @@
...
-
-import javax.servlet.http.HttpServletRequest;
-
-import static ua.com.vertex.context.SecurityWebConfig.UNKNOWN_ERROR;
 
 
 @Controller
 public class AppErrorController {
-//        implements ErrorController{
+    //        implements ErrorController{
```
Comment (backumich) [inline web/src/main/java/ua/com/vertex/controllers/exceptionHandling/AppErrorController.java]: are you sure you need it?

File: web/src/main/resources/messages.properties [hunk 3]
```diff
@@ -1,6 +1,7 @@
 typeMismatch.certificate.certificationDate=Please enter a valid date like (2011-11-11).
-typeMismatch.user.discount=This field must be between 0 and 100 including.
-typeMismatch.payment.amount=This field must be like 10,0 or 10.
-typeMismatch.course.start=Please enter a valid date like (2011-11-11T00:00:00).
-typeMismatch.course.price=The price must be up to 999999999.99.
+typeMismatch.payment.amount=This field must be like 10,0 or 10 
+typeMismatch.user.discount=This field must be between 0 and 100 including
+typeMismatch.course.start=Please enter a valid date (2001-01-01)
```
Comment (backumich) [inline web/src/main/resources/messages.properties]: what is month and what is day?

Comment (backumich) [reply web/src/main/resources/messages.properties]: you can write something like 2001-20-05 for clearness

File: web/src/main/resources/messages.properties [hunk 4]
```diff
@@ -1,6 +1,7 @@
...
+typeMismatch.course.price=The price must be up to 999999999.99
+reCaptcha.error=Wrong captcha, try again!
```
Comment (backumich) [inline web/src/main/resources/messages.properties]: Motherfucker!

File: web/src/main/java/ua/com/vertex/controllers/exceptionHandling/AppErrorController.java (hunk 2 above)
Comment (masteriza) [reply web/src/main/java/ua/com/vertex/controllers/exceptionHandling/AppErrorController.java]: I'm sorry, but this is not my code

File: web/src/main/resources/messages.properties (hunk 3 above)
Comment (masteriza) [reply web/src/main/resources/messages.properties]: I agree

File: web/src/main/resources/messages.properties (hunk 4 above)
Comment (masteriza) [reply web/src/main/resources/messages.properties]: no! This is Spaаааrtaаа!!! (=

**Pull request merged**